docs/operate/reference/components/sensor/lorawan.md

```diff
@@ -0,0 +1,519 @@
+---
+title: "Configure a LoRaWAN network"
+linkTitle: "LoRaWAN"
+weight: 60
+type: "docs"
+description: "Configure a gateway and nodes to communicate over the LoRaWAN protocol."
+tags: ["sensor", "components", "lorawan", "gateway", "node"]
+icon: true
+images: ["/icons/components/sensor.svg"]
+---
+
+[LoRaWAN (Long Range Wide Area Network)](https://lora-alliance.org/) enables sensor communication spanning kilometers with minimal power usage.
+
+To collect LoRaWAN sensor data with Viam, use the [`lorawan`](https://app.viam.com/module/viam/lorawan) {{< glossary_tooltip term_id="module" text="module" >}}. You can find the module's source code in the [lorawan GitHub repository](https://github.com/viam-modules/lorawan).
+
+## Hardware requirements
+
+- a [supported LoRaWAN gateway](https://github.com/viam-modules/lorawan?tab=readme-ov-file#lorawan-gateway-models-provided)
+- a [supported LoRaWAN node](https://github.com/viam-modules/lorawan?tab=readme-ov-file#lorawan-sensor-models-provided)
+
+## Architecture
+
+You can use Viam to create LoRaWAN networks containing one gateway and multiple nodes.
+In a LoRaWAN network, information flows in two directions:
+
+- **uplink** transmits from nodes to gateways
+- **downlink** transmits from gateways to nodes
+
+![A LoRaWAN network consisting of a single gateway and multiple nodes](/components/sensor/lorawan.png)
```

Comment:
[opt] Gets a smidge big on very wide screens; consider setting the width and adding the imgzoom class
But maybe you're getting a pretty visual from Cristina anyway?

Reply:
That's the plan. Let me know if you have any suggestions for the visual before I send in my request; I'd rather not bother Cristina more than once!